specs.json

```diff
@@ -698,6 +699,25 @@
     "required": [
     ]
   },
+  "googleOptions" : {
+    "properties": {
+      "serviceVersion": {
+        "type": "string",
+        "enum": [
+          "v1",
+          "v2"
+        ]
+      },
+      "voiceActivityTimeout": {
```

Comment:
hmm, no, we need the option to separately set each one, so as far as the application user is concerned there should be `speechStartTimeoutMs` and `speechEndTimeoutMs`

Reply:
my bad, updated